assets/styles/base.scss

```
@@ -23,6 +23,23 @@ p, ul, text {
   padding: revert;
 }
 
+.mainTOC {
+  background: var(--lightgray);
+  border-radius: 8px;
+  padding: 0.8em;
+}
```

Would like this to match padding and border-radius of the bottom cards!

```suggestion
.mainTOC {
  background: var(--lightgray);
  border-radius: 5px;
  padding: 0.75em 1em;
}
```